packages/dds/tree/src/feature-libraries/treeCursorUtils.ts

```diff
@@ -81,30 +91,24 @@ export abstract class SynchronousCursor {
  */
 class StackCursor<TNode> extends SynchronousCursor implements CursorWithNode<TNode> {
     /**
-     * Indices traversed to visit this node: does not include current level (which is stored in `index`).
+     * Might start at special root where fields are detached sequences.
+     *
+     * @param adapter - policy logic.
+     * @param indexStack - Indices traversed to visit this node: does not include current level (which is stored in `index`).
      * Even indexes are of nodes and odd indexes are for fields.
```

Comment:
This is a little confusing. Is it talking about indexes stored in the stack or the indexes of the stack?

Reply:
The use of both 'indices' and 'indexes' is a bit confusing as well. Afaik, they're the same thing, right?

Reply:
I did some rephrasing and reordering. Might be better now.